### DISCUSSION TITLE
post/frontend-sentry-monitoring/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#41] [Announcements]
# post/frontend-sentry-monitoring/

Sentry를 통해 프론트엔드에서 발생하는 오류를 신속하게 탐지하고 정확한 원인을 파악하여 빠르게 대응하는 방법을 알아봅니다.

https://tech.kakaopay.com/post/frontend-sentry-monitoring/

**Comment by myungjinki** [comment]
잘 읽었습니다. ** 임계치(threshold) ** 마크다운 에러 발견했어요.

**Comment by felixhan108** [comment]
라이프 사이클을 로깅하기 좋아하여 커스텀으로 로깅 시스템을 사용하다가 해당 서비스를 알게되어 블로그를 읽어봤네요. 
잘 봤습니다.
